sysinfo_other.go

```diff
@@ -0,0 +1,8 @@
+//go:build !linux
+// +build !linux
+
+package taskq
+
+func hasFreeSystemResources() bool {
```

Comment:
Does this function override the default taskq one?

Reply:
yes, its the only being used now actually.